component/src/main/java/org/wso2/carbon/identity/authenticator/nuxeo/NuxeoAuthenticator.java

```diff
@@ -347,32 +348,38 @@ private String getUserInfoJson(String url, String accessToken) throws Authentica
                             data.add(((JSONArray) childData).get(i).toString());
                         }
                         String result = data.toString();
-                        claims.put(ClaimMapping.build(claimsDialectUri + eachChildKey,
-                                claimsDialectUri + eachChildKey, null, false), result);
+                        fullClaimURI = buildFullClaimsURI(claimsDialectUri, eachChildKey);
+                        claims.put(ClaimMapping.build(fullClaimURI, fullClaimURI, null, false),
+                                result);
```

Comment:
Can be merged with the previous line

Reply:
addressed with https://github.com/wso2-extensions/identity-outbound-auth-nuxeo/pull/6